cockatrice/src/dlg_update.cpp

```diff
@@ -0,0 +1,114 @@
+#define HUMAN_DOWNLOAD_URL "https://bintray.com/cockatrice/Cockatrice/Cockatrice/_latestVersion"
+
+#include <QtNetwork>
```

Comment:
Instead of bundling the logic with the widget, can you define an object of something like UpdateChecker that will own the logic of spawning a thread for the update checking and downloading? Then the dialog can just connect the signals/slots and present a UI over it. This will make it easier for us to add unit tests for.